src/JavascriptVM/JavascriptVM.tsx

```diff
@@ -152,9 +202,31 @@ export const VMProvider: FunctionComponent = ({ children }) => {
             );
           }
         },
+        onCanvasCreated(canvasEl: HTMLCanvasElement) {
```

Comment:
Suggestion: Name functions based on what they do, instead of based on what they are triggered on. So in this case, maybe `init`, `initSimulator`?